src/test/java/uk/gov/ons/fwmt/csvservice/implementation/ccs/CCSConverterServiceImplTest.java

```diff
@@ -34,25 +34,30 @@
   private GatewayEventManager gatewayEventManager;
 
   @Mock
-  private CsvServiceUtils csvServiceUtils;
-
-  @Mock
-  private Storage googleCloudStorage;
+  private StorageUtils storageUtils;
 
   @Test
-  public void convertCCSCSVToCanonicalTest() throws GatewayException {
+  public void convertCCSCSVToCanonicalTest() throws GatewayException, IOException {
```

Comment:
what is this testing, no asserts or verify

Reply:
fixed